packages/ui/src/utils/UiHelpers.ts

```diff
@@ -66,6 +66,14 @@ export async function preloadImage(src: string) {
   })
 }
 
+export function roundBalance(address: string) {
+  return parseFloat(address).toFixed(2)
+}
+
+export function formatAddress(address: string) {
+  return `${address.substring(0, 5)}...${address.slice(-5)}`
+}
```

Comment:
I feel like this should have a flexible range instead of the fixed `5` but that's a nitpick

Reply:
It should be first 4 w/o the `0x` and the last 4.

Reply:
Great suggestion, will set with first 4 and last 4 as default + add in an optional range. @devceline @tunnckoCore